net.sonobus.SonoBus.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: net.sonobus.SonoBus
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '20.08'
+command: sonobus
+#rename-desktop-file: sonobus.desktop
+rename-icon: 
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=home
+# Jack
+  - --filesystem=xdg-run/pipewire-0
+  - --system-talk-name=org.freedesktop.RealtimeKit1
+
+modules:
+  - name: jack2
+    buildsystem: simple
+    build-commands:
+      - ./waf configure --prefix=/app --htmldir=/app/share/doc/jack/ --classic
+      - ./waf build -j $FLATPAK_BUILDER_N_JOBS
+      - ./waf install
+    cleanup:
+      - '*'
+    sources:
+      - type: archive
+        url: https://github.com/jackaudio/jack2/archive/refs/tags/v1.9.17.tar.gz
+        sha256: 38f674bbc57852a8eb3d9faa1f96a0912d26f7d5df14c11005ad499c8ae352f2
+
+  - name: SonoBus
+    buildsystem: simple
+    build-commands:
+      - CONFIG=Release make -j$(nproc) -C Builds/LinuxMakefile Standalone VST3
```

Comment:
I am not sure how useful it is to build the VST3 here. Nobody will see it.

A VST3 specific package could be derivated from this.

Reply:
I build VST3 from the build script from SonoBus. Are you saying I should separate it to another build module?
https://github.com/sonosaurus/sonobus/blob/main/Builds/LinuxMakefile/install.sh

Reply:
I got it to run on my system now.
I got these warnings.

```
[W][000001617.490660][context.c:224 pw_context_new()] context 0x55cab1e30fa0: can't load config jack.conf: No such file or directory. Using client.conf fallback
[E][000001617.490720][context.c:232 pw_context_new()] context 0x55cab1e30fa0: can't load client.conf config: No such file or directory
```

Reply:
what I mean about the VST3 is that building them in this flatpak is useless as no application will be able to see them.

it's somewhat a flatpak limitation, the solution will be a different flatpak, but first thing first.

The error below are actually a bug in freedestkop-sdk and nothing we can do for now. It broke Guitarix. See https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1223

Reply:
The runtime has been updated and this problem should be resolved now.